### DISCUSSION TITLE
Got error by using SDK 50.0.1.2 when build iOS Ad hoc / App Store

### DISCUSSION THREAD

**Discussion by sol-oka** [airsdk/Adobe-Runtime-Support#2298] [Q&A] [answered]
I have got this error, when using 50.0.1.2.

Undefined symbols for architecture arm64:
  "__ZN9CSWFDecen13NeedsMoreDataEPKhS1_jPj", referenced from:
      __ZN12ScriptPlayer11PushDataBufEPKhiPbb in libRuntimeHMAOT.arm-air.a(splay.o)
ld: symbol(s) not found for architecture arm64

**Comment by sol-oka** [comment]
I am using SDK 33.1.1.929 without any problem.

**Comment by ajwfrost** [comment, marked as answer]
50.0.1.2 had a missing symbol in one of the libraries, please use 50.0.1.3.
See also https://github.com/airsdk/Adobe-Runtime-Support/issues/2292

thanks

**Reply by sol-oka** [reply]
Thx. I have tested 50.0.1.3 is OK.
